### PR TITLE
Code_Review Trial Project

### PR DESCRIPTION
**Pull request by DanielAD10**
PR Description

In this PR,

Updated inline comments in code review

Thank you so much for reviewing my code review

### CODE REVIEW

**Comment by DanielAD10** [comment]
Status: REQUIRED CHANGES ♻️

_Check the comments under the review._

- [ ] Kindly make sure to update your PR title and PR description by proving a well-detailed summary and title which makes your PR clean and professional and even self-explanatory. 😉

- Example of good PR Summary
unnamed

- [ ] Kindly make sure to use the [Microverse README template](https://github.com/microverseinc/readme-template) which will help to explain your repository contents and have a clean and professional Readme. Please make sure to customize it according to your project and fill in the needed information and remember all main sections must be included except the FAQ section which is optional.
 
- [ ] Kindly make sure to add linter because of the requirements of Microverse flow.

- [ ] Kindly make sure to add text.md file and it must be presented into gitignore file and make sure that it is not represented in your repository anymore. Kindly fix that.

- In text.md file,
```
-> @@ -0,0 +1 @@
+This file should be ignored by git.
```


**Optional suggestions**

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@DanielAD10 ](https://github.com/DanielAD10) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise_**.

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @DanielAD10, 

# Requires Changes 🍀
You have done a great job so far!👏👏
Your project is almost complete. There are a few issues that you still need to work on🪲.

## Highlights
- Clear and polite comments ✅
- Comments have clear explanations why ✅


## Required Changes ♻️

- According to the project requirements, you need to have a minimum of 2 inline comments. Please update your PR to include them. You can review this [lesson](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#:~:text=Hover%20over%20the%20line%20of,comment%20field%2C%20type%20your%20comment.) as a guide

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            <label for="" id="">Leave us a comment: </label>
+            <br>
+            <textarea placeholder="Your Comment"></textarea>
+            <br>
+            <div id="submit">
+                <button>Submit</button>
+            </div>
+
+
+        </form>
```
Comment (DanielAD10) [inline index.html:64]: - [ ] Kindly make sure to delete the blank space at line 62 and 63 to look your code look professional

File: index.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <h1 id="title">Survey Form</h1>
+        <p id="description">You can share yours stories with us!</p>
+
+        <form id="survey-form">
```
Comment (DanielAD10) [inline index.html:15]: - [ ] Kindly make sure to delete the blank space at line 14 to look your code professional

File: index.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            <br>
+            <label for="dropdown" id="">Gender: </label>
+            <select id="dropdown">
+                <option value="">Choose one Gender</option>
+                <option value="Male">Male</option>
+                <option value="Female">Female</option>
+            </select>
+            <br>
+
+            <label for="state" id=""">State: </label>
```
Comment (DanielAD10) [inline index.html:33]: - [ ] Kindly make sure to delete the blank space at line 32, 39 and 46 to look your code professional

File: index.html (hunk 2 above)
Comment (DanielAD10) [inline index.html:15]: - [ ] According to the code flow of Microverse, kindly make sure to use `class` (at `h1` tag, `p` tag and `form` tag) except using `id`

File: style.css [hunk 4]
```diff
@@ -0,0 +1,60 @@
+body {
+    width: 500px;
+    height: 100%;
+    margin: 5px auto;
+    background-color: lightgray;
+}
+.container {
```
Comment (DanielAD10) [inline style.css:7]: - [ ] Kindly make sure to add a new line after the `}` to look your css code professional

File: style.css [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    height: 100px;
+    padding: 10px;
+    margin: 10px;
+}
+
+#submit {
+   text-align: center;
+   display: block;
+}
+button {
```
Comment (DanielAD10) [inline style.css:54]: - [ ] Kindly make sure to add a new line after the `}` to look your css code professional

File: style.css [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    background-color: yellow;
+    padding: 10px;
+    font-size: 16px;
+    border-radius: 10px;
+    border: none;
+}
```
Comment (DanielAD10) [inline style.css:60]: - [ ] Kindly make sure to add a new line after the` } `to look your css code professional

**Review by DanielAD10 — COMMENTED** [review]
Changes Required ♻️ 

Please make sure to check the comments below the code and do all the required changes.

Submit the re-review after you are done.

If you have any questions, just mention my name @DanielAD10 to reach out directly to me.

Thanks, 😁

File: index.html (hunk 2 above)
Comment (DanielAD10) [inline index.html:15]: - [ ]  Kindly make sure to delete the blank space at line 14 to look your code professional

Comment (DanielAD10) [inline index.html:15]: - [ ] According to the code flow of Microverse, kindly make sure to use `class` (at `h1` tag,` p` tag and` form` tag) except using` id`

File: index.html (hunk 3 above)
Comment (DanielAD10) [inline index.html:33]: - [ ] Kindly make sure to delete the blank space at line 32, 39 and 46 to look your code professional

File: index.html (hunk 1 above)
Comment (DanielAD10) [inline index.html:64]: - [ ] Kindly make sure to delete the blank space at line 62 and 63 to look your code look professional

File: index.html [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+            <br>
+            <div id="submit">
+                <button>Submit</button>
+            </div>
+
+
+        </form>
+    </div>
+    
+</body>
```
Comment (DanielAD10) [inline index.html:67]: - [ ] Kindly make sure to delete the blank space at line 66 to look your code look professional

File: style.css (hunk 4 above)
Comment (DanielAD10) [inline style.css:7]: - [ ]  Kindly make sure to add a new line after the` }` to look your css code professional

File: style.css (hunk 5 above)
Comment (DanielAD10) [inline style.css:54]: - [ ] Kindly make sure to add a new line after the` }` to look your css code professional

File: style.css (hunk 6 above)
Comment (DanielAD10) [inline style.css:60]: - [ ]  Kindly make sure to add a new line after the `}` to look your css code professional

**Review by DanielAD10 — COMMENTED** [review]
Changes Required ♻️

Please make sure to check the comments below the code and do all the required changes.

Submit the re-review after you are done.

If you have any questions, just mention my name @DanielAD10 to reach out directly to me.

Thanks, 😁

**Review by usorfaitheloho — APPROVED** [review]
Hi @DanielAD10 , 
# STATUS: APPROVED! 🥇 💯 
- [x] Used polite inline comment, Keep it up!!
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**